<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
	<meta name="keywords" content="&quot;seq.open&quot; returns error, baan,baanerp,erp,forum,discussion,bulletin board" />
	<meta name="description" content="[Archive] &quot;seq.open&quot; returns error Tools Development" />
	
	<title>&quot;seq.open&quot; returns error [Archive]  - Baanboard.com</title>
	<link rel="stylesheet" type="text/css" href="../styles.css" />
</head>
<body>
<div class="pagebody">
<div id="navbar"><a href="../index.html">Newbaanboard</a> &gt; <a href="../index.html">Baan Quick Support: Functional &amp; Technical</a> &gt; <a href="../index.html">Tools Development</a> &gt; &quot;seq.open&quot; returns error</div>
<hr />
<div class="pda"></div>

<hr />

<div class="post"><div class="posttop"><div class="username">muheeb</div><div class="date">22nd April 2004, 19:25</div></div><div class="posttext">Hi all,<br />
<br />
I face a problem with the function &quot;seq.open&quot;...<br />
<br />
I open/close for reading a file many times in my code,<br />
<br />
but in big ranges in some point &quot;seq.open&quot; fails to open<br />
<br />
for reading and returns error -11 (Internal table is full) !!<br />
<br />
Does anyone know about this error? and how can I bypass it?<br />
<br />
Thanks all.<br />
<br />
I have BaanIV running on UNIX.</div></div><hr />


<div class="post"><div class="posttop"><div class="username">JaapJD</div><div class="date">23rd April 2004, 11:50</div></div><div class="posttext">Are you sure that you close the file? There is a limitation on the number of file pointers that can be created.</div></div><hr />


<div class="post"><div class="posttop"><div class="username">muheeb</div><div class="date">23rd April 2004, 12:05</div></div><div class="posttext">Thanks you for reply,<br />
<br />
Yes I'm sure and positive 100%.<br />
By the way, the program works on one machine<br />
but in the same range in other machine failed<br />
in some point to open the file (error -11).<br />
The logic is in iteration open a file do things<br />
and then close it, to continue for the next iteration:<br />
------------------------------------------------------------------------------<br />
i = 0<br />
rep.file = seq.open(file_name1, &quot;r&quot;)<br />
while i&lt;NUM<br />
   i = i + 1<br />
   in.file = seq.open(file_name1, &quot;r&quot;)<br />
   out.file = seq.open(file_name2. &quot;w&quot;)<br />
   ...<br />
   ..<br />
  seq.close(in.file)<br />
  seq.close(out.file)<br />
endwhile<br />
seq.close(rep.file)<br />
------------------------------------------------------------------------------<br />
In some machine when NUM is big, in some point (i&gt;40) in.file returns -11 (Internal table is full), but out.file opens ok...<br />
<br />
Why this happens? <br />
is there any limitation in baan internal table? <br />
how can it be freed if it possible?<br />
<br />
Thanks.</div></div><hr />



</div>
</body>
</html>